Mutagen.Bethesda.Skyrim/Records/Assets/VoiceType/VoiceTypeAssetLookup.cs

```
@@ -0,0 +1,906 @@
+        {
+            //Don't process responses with response data
+            if (!responses.ResponseData.IsNull) continue;
+
+            //If we have selected default voices, make sure the quest voices are being checked first - they might not be part of default voices
+            var voices = GetVoices(topic, responses);
+
+            if (!questVoices.IsDefault)
+            {
+                //If we have selected default voices, make sure the quest voices are being checked first - they might not be part of default voice
+                if (voices.IsDefault)
+                {
+                    voices = (VoiceContainer) questVoices.Clone();
+                } else
+                {
+                    voices.IntersectWith(questVoices);
+                }
+            }
+
+            if (topic.Subtype == DialogTopic.SubtypeEnum.SharedInfo && _sharedInfosCache.ContainsKey(responses.FormKey))
+            {
+                var userConditions = new VoiceContainer(true);
+                foreach (var responseKey in _sharedInfosCache[responses.FormKey])
+                {
+                    var responseContext = _formLinkCache.ResolveSimpleContext<IDialogResponsesGetter>(responseKey);
+                    if (responseContext is not { Parent.Record: {} }) continue;
+
+                    var currentTopic = (IDialogTopicGetter) responseContext.Parent.Record;
+                    var currentQuest = currentTopic.Quest.TryResolve(_formLinkCache);
+                    if (currentQuest == null) continue;
+
+                    _currentQuest = currentQuest;
+                    userConditions.Insert(GetVoices(responseContext.Record.Conditions));
+                }
+
+                voices.Merge(userConditions);
+            }
+
+
+            var responseFormID = responses.FormKey.ID.ToString("X8");
+            if (voices.IsEmpty()) Console.WriteLine($"WARNING: {responseFormID} has no valid voice types {topic.EditorID}");
```

Don't want a Console write here.  This is within the context of a library, and we have no idea what system the user will be using our logic within.   Maybe it's a UI that wants to display this warning somewhere.  Or if it is a CLI, they might not want it printing this message and have to do work to filter it out somehow.

Do we need to throw an error/warning here?  We're not changing the logic either way.  Might just be appropriate to do as an [Analyzer](https://github.com/Mutagen-Modding/Mutagen.Bethesda.Analyzers)

---

That is mostly a leftover from testing so I'll just remove it.